pages/common/aria2c.md

```diff
@@ -6,7 +6,7 @@
 
 - Download a URI to a file:
 
-`aria2c {{url}}`
+`aria2c -o {{filename}} {{url}}`
```

Comment:
we like the first example without arguments, could you add this as a new example? Maybe like this:

```suggestion
`aria2c {{url}}`

Download the contents of an URL to a file:

`aria2c -o {{filename}} {{url}}`
```

Reply:
I see where you're coming from LGTM